libraries/AP_DAL/LogStructure.h

```diff
@@ -413,7 +414,7 @@ struct log_RBOH {
     { LOG_RVOH_MSG, RLOG_SIZE(RVOH),                                   \
       "RVOH", "fffIBB", "OX,OY,OZ,Del,H,Ena", "------", "------" }, \
     { LOG_ROFH_MSG, RLOG_SIZE(ROFH),                                   \
-      "ROFH", "ffffIfffB", "FX,FY,GX,GY,Tms,PX,PY,PZ,Qual", "---------", "---------" }, \
+      "ROFH", "ffffIffffB", "FX,FY,GX,GY,Tms,PX,PY,PZ,HOver,Qual", "----------", "----------" }, \
```

Comment:
maybe we should rename "HOver" "HgtOvr" if it will fit?

Reply:
Fixed